internal/engine/wazevo/backend/backend_test.go

```diff
@@ -2047,6 +2047,22 @@ L9 (SSA Block: blk6):
 	add sp, sp, #0x10
 	ldr x30, [sp], #0x10
 	ret
+`,
+		},
+		{
+			name: "VecShuffle",
```

Comment:
do you mind adding the actual call tests to  https://github.com/tetratelabs/wazero/blob/1fb5e382b19cd666356a2a7903cd749a31c16412/internal/engine/wazevo/e2e_test.go#L192-L198 as well?

Reply:
yes, I haven't pushed it yet, but I am working on it!